argilla/docs/tutorials/image_classification.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>*a field</p><p> </p><p>BTW, do you know if it is possible to make suggestions for reviewing in reviewnb as in git?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>